src/core/app.lua

```diff
@@ -283,18 +288,35 @@ function pace_breathing ()
    end
 end
 
+local function exceeded_ingress_packet_drops (app)
+   if app.ingress_packet_drops then
+      return app:ingress_packet_drops() > ingress_packet_drops.threshold
```

Comment:
given that ingress_packet_drops() should be monotonically increasing (modulo uint64 wraparound), this means that once we exceed the limit we will always exceed the limit.  probably not the right thing!